packages/grafana-ui/src/components/Monaco/CodeEditor.story.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { text } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import mdx from './CodeEditor.mdx';
+import { CodeEditor } from './CodeEditor';
+
+const getKnobs = () => {
+  return {
+    text: text('Body', 'SELECT * FROM table LIMIT 10'),
+    language: text('Language', 'sql'),
+  };
+};
+
+export default {
+  title: 'CodeEditor',
```

Comment:
Please provide a category here. Also needs to be checked with the Design System Workgroup, so maybe put this as an internal story for now.

Reply:
making this story internal for now